infrastructure/cdn-in-a-box/traffic_ops/Dockerfile

```diff
@@ -120,6 +120,9 @@ COPY infrastructure/cdn-in-a-box/enroller/server_template.json \
 	infrastructure/cdn-in-a-box/traffic_ops/tv.conf \
 	/
 
+# Generate the AES encryption key for traffic vault backend
+RUN echo $(openssl rand -base64 32) > aes.conf
```

Comment:
We should avoid storing any secrets in the built Docker image. This should be run from a script at runtime.